src/base/logger.cpp

```diff
@@ -73,7 +73,7 @@ void Logger::freeInstance()
 void Logger::addMessage(const QString &message, const Log::MsgType &type)
 {
     QWriteLocker locker(&m_lock);
-    const Log::Msg msg = {m_msgCounter++, type, QDateTime::currentMSecsSinceEpoch(), message};
+    const Log::Msg msg = {m_msgCounter++, type, QDateTime::currentSecsSinceEpoch(), message};
```

Comment:
BTW, why don't store timestamp as `QDateTime`?

Reply:
I thought about it too but I'm not certain enough to make the change.
QDateTime might have bigger (internal) data structure than `qint64`.